src/lib/ng-package/entry-point/entry-point.ts

```diff
@@ -130,10 +129,22 @@ export class NgEntryPoint {
     return this.$get('lib.flatModuleFile') || this.flattenModuleId('-');
   }
 
+  /**
+   * Returns the relative source path for secondary entry points.
+   *
+   * TODO: Compare the primary entry point base path with the base path for this entry point and rename to "relativeBasePath".
+   */
+  private get secondaryDir(): string {
```

Comment:
Nit: Since this is only used once. I prefer to to create a method for it. A better cleaner way, would also to provide the directory name, when creating the entry-point.

https://github.com/ng-packagr/ng-packagr/blob/cd2f3d54973b3a68dede9e77953e60d9bb7cc3e2/src/lib/ng-package/discover-packages.ts#L125

Reply:
Converted to a method and moved all (two) methods to the bottom of the class.

As for the code that you've linked, I would keep this as it is, because it will be removed in https://github.com/ng-packagr/ng-packagr/pull/1737.

Thanks for the time!

Reply:
This change is no longer is this MR. Sorry for all the ping-pong and the amount of changes.

I've reduced this PR to the bare minimum, updated the descrdiption am moved some changes back to #1737, which now includes just 2 commits.